thanosql/resources/_record.py

```diff
@@ -0,0 +1,21 @@
+from typing import Any, List
+
+import pandas as pd
+from pydantic import model_validator
+
+from thanosql.resources._model import BaseModel
+
+
+class Records(BaseModel):
+    records: List[dict] = []
```

Comment:
```suggestion
    data: List[dict] = []
```
I think its better to have data instead of records, just like pandas DataFrame, Series